functions/icons.zsh

```diff
@@ -290,7 +290,7 @@ case $POWERLEVEL9K_MODE in
       VCS_GIT_GITLAB_ICON            ''
       VCS_HG_ICON                    ''
       VCS_SVN_ICON                   ''
-      RUST_ICON                      ''
+      RUST_ICON                      'Rust'
```

Comment:
At some point we should agree on parenthesis or not.. But that is not the issue in this PR.